gensim/corpora/taggeddocument.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2013 Zygmunt Zając <zygmunt@fastml.com>
+# Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
+
+"""This module contains the classes of TaggedDocument and TaggedLineDocument."""
+
+from collections import namedtuple
+
+from gensim import utils
+
+
+class TaggedDocument(namedtuple('TaggedDocument', 'words tags')):
```

Comment:
`TaggedDocument` is closely-aligned with `Doc2Vec` as a description of the 'expected shape' of its training-items. It should stay in that class. 

The utility class `TaggedLineDocument` could move with other corpora-utility iterables.